pkgs/development/python-modules/stem/default.nix

```diff
@@ -1,38 +1,45 @@
-{ lib, buildPythonPackage, fetchPypi, python, mock, pythonAtLeast }:
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, cryptography
+, mock
+, python
+}:
 
 buildPythonPackage rec {
   pname = "stem";
   version = "1.8.2";
 
-  # As of May 2023, the master branch of stem contains fixes for Python 3.11
-  # that the last release (1.8.1) doesn't. The test suite fails on both master
-  # and the 1.8.1 release, so disabling rather than switching to an unstable
-  # source.
-  disabled = pythonAtLeast "3.11";
+  disabled = pythonOlder "3.6";
 
-  src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-g/sZ/9TJ+CIHwAYFFIA4n4CvIhp+R4MACu3sTjhOtYI=";
-  };
+  format = "setuptools";
```

Comment:
We can `pyproject = true;` since by default an empty pyproject.toml build-system defaults to use setuptools.